meetings/2017/05-18.md

```diff
@@ -19,11 +22,330 @@ Attendees
 - Would be good to have Flow in sooner rather than later.
   - If TypeScript definitions change if we merged it later, then it could possibly break (low risk).
 
+## 5.x recap
+
+Let's recap here major Breaking changes / changes in 5.x vs 4.x
+
+Please correct me if I'm wrong in any of those
+
+### Component reference via Constructor
+
+> isn't possible in 5.x ( Preact doesn't recognises custom-element via CTOR )
```

Comment:
@developit (I'm being lazy) are there any plans to implement this? I'd be happy to contribute it if there's nothing in the pipeline yet.